unity-renderer/Assets/Desktop/Scenes/LoginHUD/LoginHUDController.cs

```diff
@@ -40,7 +40,7 @@ private void OnPlayAsGuest()
         private void ChangeMainScene()
         {
             view.SetLoading(true);
-            SceneManager.LoadSceneAsync("MainScene", LoadSceneMode.Single);
+            SceneManager.LoadSceneAsync(sceneName: $"MainScene", LoadSceneMode.Single);
```

Comment:
`$` is redundant

Reply:
I was wondering about why Rider is complaining here. But it is actually because there is no scene with such name in the Project. What should we do here?

Reply:
extracted to https://github.com/decentraland/unity-renderer/issues/4363